holidayextras/brands
# UXUI-513 - Adding the Thomas Cook theme for Insurance Healix

#### What does this PR do? (please provide any background)
- Adds a Thomas cook theme to pattern library

#### What tests does this PR have?
None

#### How can this be tested?
You will need to clone the pattern library repo and change to branch `UXUI-513`
run `npm link ../brands/`
make sure you are on this branch in brands.
You should be able to just run pattern library using 
`npm start` and then go to `pages` and click on any of the healix pages and be happy

#### Screenshots / Screencast
<img width="797" alt="screen shot 2016-02-12 at 09 42 32" src="https://cloud.githubusercontent.com/assets/778942/13003336/fc406ace-d16c-11e5-8847-62e0b3908ea4.png">


#### What gif best describes how you feel about this work?
![Celebrate](https://media.giphy.com/media/mp1JYId8n0t3y/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
